src/anomalib/models/cfa/config.yaml

```diff
@@ -49,18 +49,20 @@ metrics:
 
 visualization:
   show_images: False # show images on the screen
-  save_images: True # save images to the file system
-  log_images: True # log images to the available loggers (if any)
-  image_save_path: null # path to which images will be saved
+  log_images: True # log images to the logger(s)
   mode: full # options: ["full", "simple"]
+  stage: test # At which stage should the images get logged. options: ["val", "test", "predict"]
```

Comment:
I just thought it would be nice to have more control over when the images are logged. But I can remove this parameter and make `test` and `predict` as the default stages for image logging.